src/proc/colorizer.cpp

```diff
@@ -241,7 +241,11 @@ namespace librealsense
             _target_stream_profile = f.get_profile().clone(RS2_STREAM_DEPTH, f.get_profile().stream_index(), RS2_FORMAT_RGB8);
 
             auto info = disparity_info::update_info_from_frame(f);
-            _depth_units = info.depth_units;
+            auto snr = ( (frame_interface *)f.get() )->get_sensor().get();
```

Comment:
So you probably not need the line `auto info = ....` anymore.

Reply:
we need it for the next line  _d2d_convert_factor = info.d2d_convert_factor;

Reply:
Can you reorder the lines as now `_d2d_convert_factor = info.d2d_convert_factor;` is separated from `auto info ...` - for clarity?